packages/portal/nuxt.config.js

```diff
@@ -476,21 +477,38 @@ export default {
       '@europeana/i18n',
       '@europeana/oembed',
       '@europeana/vue-visible-on-scroll',
+      'axios-cache-interceptor',
       'color-parse',
       'color-rgba',
       'color-space',
       'dom7',
       'ol/Collection.js',
       'ol/color.js',
+      'ol/control/Attribution.js',
+      'ol/control/Control.js',
       'ol/extent.js',
+      'ol/events.js',
       'ol/format/IIIFInfo.js',
+      'ol/geom/LineString.js',
+      'ol/interaction/DragBox.js',
       'ol/layer/Image.js',
+      'ol/layer/Layer.js',
       'ol/layer/Tile.js',
       'ol/Map.js',
-      'ol/proj/Projection.js',
+      'ol/proj.js',
+      'ol/proj/epsg3857.js',
+      'ol/render/canvas/ZIndexContext.js',
+      'ol/render/Feature.js',
+      'ol/reproj/DataTile.js',
+      'ol/reproj/Tile.js',
       'ol/source/IIIF.js',
       'ol/source/ImageStatic.js',
+      'ol/source/Source.js',
+      'ol/source/static.js',
+      'ol/source/Vector.js',
       'ol/structs/LRUCache.js',
+      'ol/style/Style.js',
+      'ol/style/RegularShape.js',
```

Comment:
Are these all needed? I don't see them used directly, or are they dependencies for other OpenLayers modules we are using?